File: solutions/119.java
class Solution {
    public List<Integer> getRow(int rowIndex) {
        List<Integer> pascalRow = new ArrayList<>();
        long res = 1;
        pascalRow.add((int)res);
        for (int col = 1; col < rowIndex + 1; col++) {
            res = res * (rowIndex + 1 - col)/col;
            pascalRow.add((int)res);
        }
        return pascalRow;
    }
}

File: solutions/119.py
class Solution:
    def getRow(self, rowIndex: int) -> List[int]:
        pascal_row = []
        res = 1
        pascal_row.append(res)
        for col in range(1, rowIndex + 1):
            res = res * (rowIndex + 1 - col) // col
            pascal_row.append(res)
        return pascal_row